src/chip.cs

```diff
@@ -1764,4 +1799,706 @@ interface ChipQRCodeSetupPayloadParser
+		void ReadAttributeTvChannelLineup (ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("readAttributeCurrentTvChannelWithResponseHandler:")]
+		void ReadAttributeCurrentTvChannel (ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("readAttributeClusterRevisionWithResponseHandler:")]
+		void ReadAttributeClusterRevision (ChipResponseHandler responseHandler);
+	}
+
+	[Mac (12,0), Watch (8,0), TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (ChipCluster), Name="CHIPWakeOnLan")]
+	[DisableDefaultCtor]
+	interface ChipWakeOnLan
+	{
+
+		[Export ("initWithDevice:endpoint:queue:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (ChipDevice device, byte endpoint, DispatchQueue queue);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("readAttributeWakeOnLanMacAddressWithResponseHandler:")]
+		void ReadAttributeWakeOnLanMacAddress(ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("readAttributeClusterRevisionWithResponseHandler:")]
+		void ReadAttributeClusterRevision (ChipResponseHandler responseHandler);
+	}
+
+	[Mac (12,0), Watch (8,0), TV (15,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof (ChipCluster), Name="CHIPWindowCovering")]
+	[DisableDefaultCtor]
+	interface ChipWindowCovering
+	{
+
+		[Export ("initWithDevice:endpoint:queue:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (ChipDevice device, byte endpoint, DispatchQueue queue);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("windowCoveringDownClose:")]
+		void WindowCoveringDownClose (ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("windowCoveringGoToLiftPercentage:responseHandler:")]
+		void WindowCoveringGoToLiftPercentage (byte percentageLiftValue, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("windowCoveringGoToLiftValue:responseHandler:")]
+		void WindowCoveringGoToLiftValue (ushort liftValue, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("windowCoveringGoToTiltPercentage:responseHandler:")]
+		void WindowCoveringGoToTiltPercentage (byte percentageTiltValue, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("windowCoveringGoToTiltValue:responseHandler:")]
+		void WindowCoveringGoToTiltValue (ushort tiltValue, ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("windowCoveringStop:")]
+		void WindowCoveringStop (ChipResponseHandler responseHandler);
+
+		[Async (ResultTypeName = "ChipReadAttributeResult")]
+		[Export ("windowCoveringUpOpen:")]
+		void WindowCoveringUpOpen (ChipResponseHandler responseHandler);
```

Comment:
I think you can (maybe?) get rid of the `WindowCovering` prefix on all of the above